sdk/identity/azure-identity/azure/identity/_version.py

```diff
@@ -2,4 +2,4 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-VERSION = "1.1.0"
+VERSION = '1.1.1'
```

Comment:
Please keep the double quotes so our formatting tool (black) doesn't revert the change.

Reply:
@chlowell  Done. Version was updated by version increment tool. I have fixed it manually for now. will submit another PR to fix this in tool.